src/main/java/seedu/address/logic/parser/ShiftCommandParser.java

```diff
@@ -24,9 +24,10 @@ public ShiftCommand parse(String args) throws ParseException {
 
         requireNonNull(args);
         try {
-            String[] splitArgs = args.trim().split(" ");
+            String[] splitArgs = args.trim().split(" ", 2);
```

Comment:
This only allows for splitting from the first space. So any preceding spaces will be in the second argument, and it will be picked up by `parseShiftDays`.